api/bots_api/bot_lib.py

```diff
@@ -105,6 +111,67 @@ def get_config_info(self, bot_name, section=None):
         config.readfp(open(conf_file_path))  # type: ignore
         return dict(config.items(section))
 
+class EmbeddedBotHandler(object):
+    def __init__(self, client):
```

Comment:
Would it make sense to pass in the `user_profile` directly instead of retrieving it using a `Client`?  I think one of the goals of the `EmbeddedBotHandler` is to not require a `Client`.